vendor/github.com/intel/govmm/qemu/qemu.go

```diff
@@ -1126,6 +1126,10 @@ type Memory struct {
 	// MaxMem is the maximum amount of memory that can be made available
 	// to the guest through e.g. hot pluggable memory.
 	MaxMem string
+
```

Comment:
In the patch title - `hack/vendor` made me think
- that this was a patch in the https://github.com/kata-containers/runtime/tree/master/virtcontainers/hack directory
- and reminded me that we should have changed that subdir name from `hack` when we moved over VC
- and that no PRs should probably container the word `hack` ;-)

Can you drop the `hack` from the subject pls?

Reply:
That commit will be reworked once the govmm PR gets merged. The `hack` in the commit subject is to remind myself of the rework ;)